build/mvn

```diff
@@ -22,7 +22,7 @@ _DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # Preserve the calling directory
 _CALLING_DIR="$(pwd)"
 # Options used during compilation
-_COMPILE_JVM_OPTS="-Xmx2g -XX:MaxPermSize=512M -XX:ReservedCodeCacheSize=512m"
```

Comment:
It's still in Java 8. I actually removed this because I think it's defunct and no longer needed, but I admit it's not strictly related to Java 8. I'd put it back if anyone has doubts, but if nobody can recall what it's for (I've never set it in dev or production) maybe it's removable now